notebooks/contrib-dev/McCabe_Thiele_Steven_Final.ipynb

```diff
@@ -0,0 +1,1665 @@
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "### BEGIN SOLUTIONS\n",
+        "\n",
+        "# slope and y-intercept of the 45 degree line\n",
+        "slope = 1\n",
+        "y_intercept = 0\n",
+        "Efficiency = 1\n",
+        "\n",
+        "# call stair funciton to generate the points needed to plot\n",
+        "complete_reflux = stair(slope=1, y_intercept=0, x_start=xB, y_start=xB, y_end=xD)\n",
+        "# extract x and y points for the plot\n",
+        "xplot = complete_reflux[0]\n",
+        "yplot = complete_reflux[1]\n",
+        "total_stage = complete_reflux[2]\n",
+        "print(\"The number of total theoretical stage is \", total_stage)\n",
+        "\n",
+        "# McCabe-Thiele diagram\n",
+        "fig = plt.figure(figsize=(4, 4))\n",
+        "# plot the LVE line\n",
+        "plt.plot(liqvapx, liqvapy)\n",
+        "# plot the 45 degree line\n",
+        "plt.plot([0, 1], [0, 1], color=\"orange\", linestyle=\":\")\n",
+        "# plot the VLE line\n",
+        "plt.plot(xplot, yplot, color=\"red\")\n",
+        "plt.xlabel(\"Liquid Mole Fraction \\n of Methanol\", fontsize=10)\n",
+        "plt.ylabel(\"Vapor Mole Fraction \\n of Methanol\", fontsize=10)\n",
+        "plt.xticks(fontsize=15)\n",
+        "plt.yticks(fontsize=15)\n",
+        "plt.tick_params(direction=\"in\", top=True, right=True)\n",
+        "plt.title(\"McCabe-Thiele of a total \\n reflux system\", fontsize=10)\n",
+        "plt.legend(\n",
+        "    labels=(\"LVE line\", \"45 line\", \"step line\"),\n",
```

Comment:
Please add a degree symbol after 45. Also make sure this is defined in the background text.